packages/dashboard/src/msw/useWorker.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+import { worker } from './browser';
+
+export const useWorker = () => {
+  useEffect(() => {
+    worker.start();
+
+    return () => {
+      worker.stop();
+    };
```

Comment:
do you think its worth adding something like location.reload() here to force a memory flush ?

Reply:
The problem is that the tanstack client is cached across story renders, I don't like the idea of putting a patch for that with our mock service worker code because it ties them together in a weird way.